lambda_function.py

```diff
@@ -0,0 +1,50 @@
+import json
+import os
+from google.cloud import bigquery
+from google.oauth2 import service_account
+
+def lambda_handler(event, context):
+    messageBody = event["Records"]
+
+    for eachMessage in messageBody:
+        message = json.loads(eachMessage["Sns"]["Message"])[0]
+        
+        row = {}
+        
+        row["DateTime"] = message["DateTime"]
```

Comment:
This is JSON, no? make everything lowercase, separateed by underscore. @deepansh96 can say what case it is. kebab case, maybe? :D